padertorch/contrib/examples/tasnet/train.py

```diff
@@ -12,6 +12,8 @@
 from sacred import Experiment
 import sacred.commands
 
+# sacred.SETTINGS.CONFIG.READ_ONLY_CONFIG = False
+
```

Comment:
Is this comment necessary?

Reply:
Not really. I added it, because I expected that the script fails without disable `READ_ONLY_CONFIG`.
Someone reported that the `betas` in the Adam optimizer caused some problems.
But this may only fail in some pytorch versions.

I will activate that line, to prevent some strange error messages from sacred.